### PR TITLE
Feature/base components

### PR DESCRIPTION
**Pull request by Katarzyna-Kadziolka**

### CODE REVIEW

File: doskvol-tools/src/components/common/baseTable/BaseTable.vue [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+// const randomRow = b.rows.find(x => randomDiceResult <= x.diceResult.to);
+// if (randomRow === undefined) throw new Error("HELP");
+// const randomValue = randomRow.values[randomInteger(0, randomRow.values.length - 1)]
+//to będzie logika do random
+//TODO przenieść do odpowiedniego pliku
+const formatDiceResult = (from: number, to: number): string => {
+  if (from === to) return from.toString();
+  return `${from}-${to}`;
+}
+const chosenTd = ref("");
```
Comment (Morasiu) [inline doskvol-tools/src/components/common/baseTable/BaseTable.vue:30]: `chosedTd`? Maybe chosenTableSection would be a better name.

File: doskvol-tools/src/components/generator/streetGenerator/StreetGenerator.vue [hunk 2]
```diff
@@ -1,8 +1,96 @@
...
+  {
+    label: "Pospolity",
+    value: "common"
+  },
+  {
+    label: "Rzadki",
+    value: "rare"
+  }
+]
+const changes = ref("")
```
Comment (Morasiu) [inline doskvol-tools/src/components/generator/streetGenerator/StreetGenerator.vue:19]: `changes`?

File: doskvol-tools/src/composables/UniqueID.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+let UUID = 0
```
Comment (Morasiu) [inline doskvol-tools/src/composables/UniqueID.ts:1]: Do you prefer own function or just using lodash?

File: doskvol-tools/src/components/generator/streetGenerator/StreetGenerator.vue [hunk 4]
```diff
@@ -1,8 +1,96 @@
...
+        "cośCB",
+        "cośDB",
+        "cośEB",
+        "cośFB",
+        "coGB",
+      ]
+    },
+  ]
+}
+const chosenValue = ref("")
```
Comment (Morasiu) [inline doskvol-tools/src/components/generator/streetGenerator/StreetGenerator.vue:71]: Should this be modelValue and v-model?

**Review by Morasiu — CHANGES_REQUESTED** [review]

File: doskvol-tools/src/components/common/baseTable/BaseTable.vue (hunk 1 above)
Comment (Katarzyna-Kadziolka) [reply doskvol-tools/src/components/common/baseTable/BaseTable.vue]: Ok, I changed it

File: doskvol-tools/src/components/generator/streetGenerator/StreetGenerator.vue (hunk 2 above)
Comment (Katarzyna-Kadziolka) [reply doskvol-tools/src/components/generator/streetGenerator/StreetGenerator.vue:19]: This is only for check component work. In next step I will create object like Street and I will bind it with props from this object

File: doskvol-tools/src/components/generator/streetGenerator/StreetGenerator.vue (hunk 4 above)
Comment (Katarzyna-Kadziolka) [reply doskvol-tools/src/components/generator/streetGenerator/StreetGenerator.vue:71]: I can and v-model to td in table. Do you mind something else?

**Pull request merged**